examples/javascript-vite/src/main.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * Main application entry file for Nutrient Web SDK with Vite.
+ *
+ * This example supports both CDN and npm package loading approaches:
+ * - CDN: Uses global window.NutrientViewer (default)
+ * - npm package: Uses ES6 import (when USE_NPM=true)
+ *
+ * Features:
+ * - Drag and drop PDF loading
+ * - File picker for PDF selection
+ * - Tracks unsaved annotations
+ * - Loads a default sample PDF on page load
+ */
+
+import dragDrop from "drag-drop";
```

Comment:
We can simplify this, drag and drop, file selection are not needed, just a simple example that loads the remote PDF (similar to react guide) should be fine.